src/components/editMeasure/details/measureInformation/MeasureInformation.test.tsx

```diff
@@ -131,9 +131,7 @@ jest.mock("@madie/madie-util", () => ({
     getAccessToken: () => "test.jwt",
   })),
   useKeyPress: jest.fn(() => false),
-  useFeatureFlags: jest.fn().mockReturnValue({
-    generateCMSID: true,
-  }),
+  useFeatureFlags: jest.fn().mockReturnValue({}),
```

Comment:
this can be removed and the corresponding usage from the component- `const featureFlags = useFeatureFlags();`